conda/cmake/RobotologySuperbuildGenerateCondaRecipes.cmake

```diff
@@ -142,7 +142,7 @@ macro(generate_metametadata_file)
 endmacro()
 
 macro(generate_conda_recipes)
-  set(python_generation_script "${CMAKE_CURRENT_SOURCE_DIR}/conda/python/generate_conda_recipes_from_metadatadata.py")
+  set(python_generation_script "${CMAKE_CURRENT_SOURCE_DIR}/conda/python/generate_conda_recipes_from_metametadata.py")
```

Comment:
Just a curiosity, why is it called `metametadata`?  Curious about the two metas ;D

Reply:
A conda recipe contains a file `meta.yaml` that is supposed to contain metadata of the package being built (see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html). In our case, this meta.yaml file is generated from a template (https://github.com/robotology/robotology-superbuild/blob/master/conda/recipe_template/meta.yaml) using [Jinja2](https://jinja.palletsprojects.com/en/2.11.x/) template engine.
This template so is used to generate the `meta.yaml` file that contains **meta**data, but it does so by getting the relevant info that so contains **metametadata**, because it is metadata about the metadata. This **memetametadata** file itself is generated automatically by the CMake logic of the superbuild, for more on this feel free to refer to https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-recipe-generation.md#internals . Actually I noticed a typo also there.